Student/Student/Submissions/SubmissionRubric/RubricViewController.swift

```diff
@@ -151,7 +151,9 @@ class RubricCollectionViewCell: UICollectionViewCell {
     }
 
     static func computedHeight(rubric: RubricViewModel, containerFrame: CGRect) -> CGFloat {
-        let otherViewHeights: CGFloat = 111
+        let spaceToCollapse: CGFloat = rubric.longDescription.count == 0 ? 16 : 0
+        let otherViewHeights: CGFloat = 111 - spaceToCollapse
```

Comment:
This may be a knit pick so ignore if you disagree.  This type of code (this method, not your additions) is sometimes messy to make changes to if not the original author and familiar with how all the heights are calculated.  I wonder if we can be more a little more intentional and specific about what heights are being combined for the next person who comes along and makes changes here. 

```
let otherViewHeights: CGFloat = 96.0
...
return otherViewHeights + circles + comment + longDescriptionButton
```

Reply:
Oh yeah that makes a lot of sense.